aten/src/ATen/native/mps/operations/BinaryOps.mm

```diff
@@ -21,7 +21,7 @@
 typedef MPSGraphTensor* (^BinaryOpBlock)(MPSGraph*, MPSGraphTensor*, MPSGraphTensor*);
 #define BinaryOpFn() MPSGraphTensor* (MPSGraph* mpsGraph, MPSGraphTensor* primary, MPSGraphTensor* secondary)
 
-void binaryOpTensor(const Tensor& self_t, const Tensor& other_t, const Tensor& output, std::string op_name, BinaryOpBlock binaryBlock)
+void binaryOpTensor(const Tensor& self_t, const Tensor& other_t, const Tensor& output, std::string op_name, bool is_arithmetic, BinaryOpBlock binaryBlock)
```

Comment:
Could you put a comment about what problem is this patch trying to fix?

Reply:
Sure, thanks for pointing that out

Reply:
Added a comment